gatsby-node.js

```diff
@@ -110,16 +110,3 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
   })
 
 }
-
-
-/*
-Webpack Config
-================================================================================
-*/
-exports.onCreateWebpackConfig = ({ stage, actions }) => {
```

Comment:
I removed this custom configuration because it didn't play well with TypeScript. We could set up path aliases in a future PR to get the same effect, though.

Reply:
Yes!  Move fast n fix things.  :sunglasses: